examples/preapproval/search/search_subscription_payment_orders.rb

```diff
@@ -12,11 +12,39 @@
 report = PagSeguro::SubscriptionSearchPaymentOrders.new(code, status, options)
 
 if report.valid?
+  puts "====== Report ======"
 
   while report.next_page?
     report.next_page!
 
-    puts report.payment_orders.inspect
+    puts "=> Page #{report.page}/#{report.total_pages}"
+    report.payment_orders.each_with_index do |order, i|
```

Comment:
It is correct, but I prefer to use `index` instead of `i`. I think `i` means nothing, but `index` means :ok_hand: